src/cobra/io/mat.py

```diff
@@ -856,7 +903,9 @@ def from_mat_struct(
         rxn_group_names = set(rxn_subsystems).difference({None})
         new_groups = []
         for g_name in sorted(rxn_group_names):
-            group_members = model.reactions.query(lambda x: x.subsystem == g_name)
+            group_members = model.reactions.query(
+                lambda x, g_n=g_name: x.subsystem == g_n
```

Comment:
I don't think it makes sense to define default values for a lambda function. The previous one looks better to me.

Reply:
Okay. My linter complained about it. Reverted.